Fix kmod panic when VIMAGE is disabled in the kernel and port

Submitted by:	Gleb Kurtsou <gleb@freebsd.org>
--- src/VBox/HostDrivers/VBoxNetAdp/Makefile.kmk.orig	2013-04-12 03:38:10.000000000 -0700
+++ src/VBox/HostDrivers/VBoxNetAdp/Makefile.kmk	2013-06-11 21:24:05.000000000 -0700
@@ -143,12 +143,15 @@
 		$$(if $$(eq $$(VBoxNetAdp/freebsd/Makefile_VBOX_HARDENED),$$(VBOX_WITH_HARDENING)),,FORCE) \
 		| $$(dir $$@)
 	$(QUIET)$(RM) -f -- $@
- ifndef VBOX_WITH_HARDENING
-	$(QUIET)$(SED) -e "s;-DVBOX_WITH_HARDENING;;g" --output $@ $<
- else
 	$(QUIET)$(CP) -f $< $@
+ ifndef VBOX_WITH_HARDENING
+	$(QUIET)$(SED) -e "s;-DVBOX_WITH_HARDENING;;g" --output $@.tmp $@
+	${QUIET}$(MV) -f $@.tmp $@
+ endif
+ ifndef VBOX_WITH_NETFLT_VIMAGE
+	$(QUIET)$(SED) -e "s;-DVIMAGE;;g" --output $@.tmp $@
+	${QUIET}$(MV) -f $@.tmp $@
  endif
-
 endif # freebsd
 
 include $(FILE_KBUILD_SUB_FOOTER)
